common/static/js/vendor/jquery.leanModal.js

```diff
@@ -0,0 +1,61 @@
+// leanModal v1.1 by Ray Stone - http://finelysliced.com.au
+// Dual licensed under the MIT and GPL
+
+// Updated to prevent divs with duplicate IDs from being rendered.
+
+(function($) {
+    $.fn.extend({
+        leanModal: function(options) {
+            var defaults = {
+                top: 100,
+                overlay: 0.5,
+                closeButton: null
+            };
+
+            // Only append the overlay element if it isn't already present.
+            if ($("#lean_overlay").length == 0) {
```

Comment:
I checked.  It returns an empty array if the element isn't present, and only appends if the length of the array is equal to 0.  So anytime it does exist, it skips the part where the element is added.